lib/net.c

```diff
@@ -1913,10 +2031,10 @@ start_iface(struct gatekeeper_if *iface, unsigned int num_attempts_link_get)
 		goto stop;
 	}
 
-
```

Comment:
This dropped line must go to patch **lib/net: handle bonded interfaces as any other**